packages/core/admin/server/middlewares/rateLimit.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+const { toLower } = require('lodash/fp');
+
+const { RateLimitError } = utils.errors;
+
+module.exports =
+  (config, { strapi }) =>
+  async (ctx, next) => {
+    const ratelimit = require('koa2-ratelimit').RateLimit;
+
+    const userEmail = toLower(ctx.request.body.email) || 'unknownEmail';
+
+    return ratelimit.middleware({
+      interval: { min: 5 },
+      max: 5,
+      prefixKey: `${userEmail}:${ctx.request.path}:${ctx.request.ip}`,
```

Comment:
If someone knows the IP address(es) of an admin, their IP address could be spoofed to DoS someone out of their admin account. Is that a valid concern here?

My personal opinion is that it seems rare enough that we don't really have to worry about it, as long as there's an option to disable the ratelimiter in those rare cases someone does have the problem (hence my concern in my previous comment)

Reply:
It would be a very niche concern I think and practically impossible to work around without disabling the limiter. It would be faster and easier to simply DDoS the entire application than just the auth endpoint.

Reply:
Yeah, my thoughts too. As long as there's a way to safely disable the ratelimiter, that's good enough for me.